.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
- [ ] It would be better if we use the Microverse recommended linter file, [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml), this file is used by all the students of Microverse as a way of enforcing code standard

Reply:
That's where mine is from, and it's inside the .github/workflows folder, the one you pointed out (eslintrc.json) is the local linters, from here https://github.com/microverseinc/linters-config/tree/master/javascript. I'm a little confused by what you mean.

Reply:
Addressed this